tox.ini

```diff
@@ -110,7 +110,8 @@ commands =
 
 [testenv:py3-pymongo-latest]
 commands =
-    pip install git+https://github.com/mongodb/mongo-python-driver.git@master#egg=pymongo[encryption]
+    pip install git+https://github.com/mongodb/mongo-python-driver.git@master
+    pip install --pre pymongocrypt
```

Comment:
I'm curious, why did you decide to use `--pre` even though we didn't publish a prerelease?

Reply:
I had mentioned in the ticket that this would allow us to publish a beta and consume it here.

Reply:
(in the future)

Reply:
SGTM